sched/group/group_create.c

```diff
@@ -222,5 +238,6 @@ void group_postinitialize(FAR struct task_tcb_s *tcb)
    * task has exited.
    */
 
-  group->tg_pid = tcb->cmn.pid;
+  group->tg_pid = (group == &g_kthread_group) ? group->tg_pid :
```

Comment:
why change

Reply:
The g_kthread_group is shared among all kthreads and its tg_pid shall not be set to the pid of each kthread, right?

Reply:
The latest updtae used `if` for this.